src/main/java/uk/gov/hmcts/reform/preapi/controllers/CaseController.java

```diff
@@ -126,4 +126,11 @@ public ResponseEntity<Void> undeleteCase(@PathVariable UUID id) {
         caseService.undelete(id);
         return ResponseEntity.ok().build();
     }
+
+    @PostMapping("/close-pending")
```

Comment:
Is there a better way to trigger this functionality? 

I'm assuming the plan is to trigger this from a scheduled job from a flow. There should be a way to bake this directly into the API to avoid unnecessary calls to the API. 

Maybe we could make use of the `@PostLoad` entity life cycle event or the spring scheduler using `@Scheduled`